test/util/shared_index_informer.go

```diff
@@ -26,3 +27,4 @@ func (s *SharedIndexInformer) LastSyncResourceVersion() string
 func (s *SharedIndexInformer) AddIndexers(cache.Indexers) error                           { return nil }
 func (s *SharedIndexInformer) GetIndexer() cache.Indexer                                  { return s.Indexer }
 func (s *SharedIndexInformer) SetWatchErrorHandler(handler cache.WatchErrorHandler) error { return nil }
+func (s *SharedIndexInformer) SetTransform(handler cache.TransformFunc) error             { panic("implement me") }
```

Comment:
this was a new function added to the `SharedIndexInformer` interface in k8s client-go v0.24.3

Reply:
maybe return nil?